Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine
+
+# Setup apt
+RUN apk -U --no-cache add \
+               build-base \
+               git \
+               linux-headers \
+               python3 \
+               python3-dev && \ 
+
+# Setup Snare
+    git clone --depth=1 https://github.com/mushorg/snare /opt/snare && \
+    cd /opt/snare/ && \
+    pip3 install --no-cache-dir --upgrade pip setuptools && \
+    pip3 install --no-cache-dir -r requirements.txt && \
+    python3.6 clone.py --target http://example.com && \
+    
+# Clean up
+    apk del --purge \
+            build-base \
+            linux-headers \
+            python3-dev && \
+    rm -rf /root/* && \
+    rm -rf /tmp/* /var/tmp/* && \
+    rm -rf /var/cache/apk/*
+
+# Start snare
+WORKDIR /opt/snare
+CMD /usr/bin/python3.6 /opt/snare/snare.py --no-dorks --auto-update false --host-ip 0.0.0.0 --port 80 --page-dir example.com
```

Comment:
You've forgotten EXPOSE

Reply:
I intentionally left it out and I have never used `EXPOSE` for my production `Dockerfiles`. It simply is not necessary (you use `ports` in `docker-compose.yml` or `docker run -p <src_port>:<dest_port>` for correct startup) and it even messes with things should a user decide to build an image which is based (`FROM`) on the one built with this `Dockerfile` (if published on a registry).

Reply:
But it is better to use it in a common dockerfile, isn't it? 
If I use some custom logic based on EXPOSE. For example, nginx-proxy docker image is working with EXPOSE port to proxy it.